src/AppBundle/Service/Mailer/MailFactory.php

```diff
@@ -311,8 +307,7 @@ public function createReportSubmissionConfirmationEmail(EntityDir\User $user, En
      */
     public function createPaReportSubmissionConfirmationEmail(EntityDir\User $user, EntityDir\Report\Report $submittedReport, EntityDir\Report\Report $newReport, $pdfBinaryContent)
```

Comment:
This method is now only calling another method with the same parameters. If it's wanted (Pa=Lay, not sure), I'd replace the usage of createPaReportSubmissionConfirmationEmail with createReportSubmissionConfirmationEmail and remove a method overall